server/app/views/admin/programs/ProgramBaseView.java

```diff
@@ -36,10 +37,32 @@ public enum ProgramHeaderButton {
     EDIT_PROGRAM,
     /** Redirects to the program details editing page. */
     EDIT_PROGRAM_DETAILS,
+    /** Redirects to the program image editing page. */
+    EDIT_PROGRAM_IMAGE,
     /** Redirects to previewing this program as an applicant. */
     PREVIEW_AS_APPLICANT,
   }
 
+  /**
+   * Returns the header buttons used for editing various parts of the program (details, image,
+   * etc.).
+   *
+   * @param isEditingAllowed true if the program currently allows editing and false otherwise.
```

Comment:
question/suggestion (blocking): Should this be "true if the view allows editing"? Please add more details in this comment about when this is true (Eg. true when the program is in draft mode)

Reply:
Added more details!